### PR TITLE
Remove Facebook Orders sync functionality

### PR DESCRIPTION
**Pull request by message-dimke**
### Changes proposed in this Pull Request:

Removing Facebook Orders sync functionality as the one which was not functional and mentions of which were already removed from the documentation. 

### Changelog entry

> Tweak - Remove Facebook Orders sync functionality

### CODE REVIEW

**Comment by budzanowski** [comment]
What about `cancel_order` from https://github.com/woocommerce/facebook-for-woocommerce/blob/97ef8086a11f0a3ace0408087f49cc02cea0feea/includes/API.php#L636

can it also be removed?

**Comment by budzanowski** [comment]
The same question for `acknowledge_order` from:

https://github.com/woocommerce/facebook-for-woocommerce/blob/97ef8086a11f0a3ace0408087f49cc02cea0feea/includes/API.php#L618

**Comment by budzanowski** [comment]
I wonder how the connection flow changes when we don't request `pages_read_engagement` .

**Comment by budzanowski** [comment]
> Is the pages_read_engagement scope exclusively for order management or does it have other purposes also?

It looks like it has nothing to do with the orders https://developers.facebook.com/docs/permissions/reference/pages_read_engagement/ 😕

File: includes/API.php [hunk 1]
```diff
@@ -13,7 +13,6 @@
 
 defined( 'ABSPATH' ) or exit;
 
-use WooCommerce\Facebook\API\Orders\Order;
```
[the remark this answers is not in the document]
Comment (message-dimke) [reply includes/API.php:16]: @ibndawood , order references inside `facebook-commerce-events-tracker.php` seems to be all related to WooCommerce Orders not Facebook Orders, so will keep it. Removed `Shipment` class as the one not needed containing the mapping to Facebook shipment options.

**Pull request merged**